src/common/ITEMS.js

```diff
@@ -34,6 +34,13 @@ const ITEMS = {
     icon: 'inv_misc_necklace15',
     quality: ITEM_QUALITIES.LEGENDARY,
   },
+  ARCHIMONDES_HATRED_REBORN: {
+    id: 144249,
+    absorbId: 235169,
```

Comment:
Please add this as a spell instead. I realize it results in little bit more work but it pays off for reducing duplication, improving consistency and making everything more predictable.